sshfs-win.c

```diff
@@ -314,6 +321,16 @@ static int do_svc(int argc, char *argv[])
         sshfs, SSHFS_ARGS, idmap, authmeth, volpfx, portopt, remote, argv[2], 0,
     };
 
+    if (strlen(portopt) == 0)
```

Comment:
Nitpick: Consider changing the test `strlen(portopt) == 0` to something like `'\0' == portopt[0]`.